src/EventStore.Client.PersistentSubscriptions/EventStorePersistentSubscriptionsClient.Info.cs

```diff
@@ -0,0 +1,88 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using EventStore.Client.PersistentSubscriptions;
+using Grpc.Core;
+
+#nullable enable
+namespace EventStore.Client {
+	partial class EventStorePersistentSubscriptionsClient {
+		/// <summary>
+		/// Gets the status of a persistent subscription to $all
+		/// </summary>
+		/// <param name="groupName"></param>
+		/// <param name="deadline"></param>
+		/// <param name="userCredentials"></param>
+		/// <param name="cancellationToken"></param>
+		/// <returns></returns>
+		public async Task<PersistentSubscriptionInfo> GetInfoToAllAsync(string groupName, TimeSpan? deadline = null,
+			UserCredentials? userCredentials = null, CancellationToken cancellationToken = default) {
+
+			var channelInfo = await GetChannelInfo(cancellationToken).ConfigureAwait(false);
+			if (channelInfo.ServerCapabilities.SupportsPersistentSubscriptionsGetInfo) {
```

Comment:
just a matter of style, but personally i would invert this if to reduce nesting since its essentially a guard